api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1112,6 +1114,16 @@ type FoundationDBClusterAutomationOptions struct {
 	// UseNonBlockingExcludes defines whether the operator is allowed to use non blocking exclude commands.
 	// The default is false.
 	UseNonBlockingExcludes *bool `json:"useNonBlockingExcludes,omitempty"`
+
+	// MaxConcurrentMisconfiguredReplacements defines how many process groups can be concurrently
+	// replaced if they are misconfigured. If the value will be set to 0 this will block replacements
+	// and these misconfigured Pods must be replaced manually or by another process. For each reconcile
+	// loop the operator calculates the maximum number of possible replacements by taken this value as the
+	// upper limit and removes all ongoing replacements that have not finished. Which means if the value is
+	// set to 5 and we have 4 ongoing replacements (process groups marked with remove but not excluded) the
+	// operator is allowed to replace on further process group.
+	// +kubebuilder:validation:Minimum=0
+	MaxConcurrentMisconfiguredReplacements *int `json:"maxConcurrentMisconfiguredReplacements,omitempty"`
```

Comment:
As you alluded to in the PR body, I think this name is going to be confusing to people. We could call them "discretionary" replacements, or maybe we could just call this `MaxConcurrentReplacements` and call out in the documentation that it only applies to replacements that the operator chooses to do.